### PR TITLE
Normalize sql carbon tag across all files

### PR DESCRIPTION
**Pull request by abist**
Recently while working on something I noticed the format of the `// {{SQL CARBON EDIT}}` tag was all over the place in different files. Normalized it to one format (with least number of changes) so it's easier to search.

### CODE REVIEW

**Review by kburtram — APPROVED** [review]
I suspect this is going to cause a bunch of merge conflicts in the bot 😄

**Comment by abist** [comment]
@kburtram You were right, probably not the best time for this PR 😅 , will probably wait post merge to merge this PR.

**Comment by kburtram** [comment]
@abist whatever's easier.  the conflicts will likely be easy to resolve.

File: extensions/notebook/yarn.lock [hunk 1]
```diff
@@ -1756,7 +1756,7 @@ uri-js@^4.2.2:
   dependencies:
     punycode "^2.1.0"
 
-url-parse@^1.5.0, url-parse@~1.4.3:
+url-parse@^1.5.1, url-parse@~1.4.3:
```
Comment (Charles-Gagnon) [inline extensions/notebook/yarn.lock:1759]: This change doesn't seem related

File: remote/yarn.lock [hunk 2]
```diff
@@ -349,19 +349,6 @@ graceful-fs@^4.1.2, graceful-fs@^4.1.6:
   resolved "https://registry.yarnpkg.com/graceful-fs/-/graceful-fs-4.2.4.tgz#2256bde14d3632958c465ebc96dc467ca07a29fb"
   integrity sha512-WjKPNJF79dtJAVniUlGGWHYGz2jWxT6VhN/4m1NdkbZ2nOsEF+cI1Edgql5zCRhs/VsQYRvrXctxktVXZUkixw==
 
-har-schema@^2.0.0:
```
Comment (Charles-Gagnon) [inline remote/yarn.lock:352]: Merge in latest from main - I just fixed this there

**Review by Charles-Gagnon — APPROVED** [review]

**Review by alanrenmsft — APPROVED** [review]

**Comment by Charles-Gagnon** [comment]
@abist Why did you leave in the yarn.lock change? It's not a huge deal but typically we shouldn't be including unrelated changes in PRs - it makes it harder to track things down if we need to look at why something changed.

**Comment by abist** [comment]
@Charles-Gagnon I didn't introduce it, it came in from the main merge if it was there

**Comment by Charles-Gagnon** [comment]
mm, I pointed it out before you merged in from main. So don't think that's the case

**Comment by Charles-Gagnon** [comment]
Your original commit has the change in it : https://github.com/microsoft/azuredatastudio/pull/14445/commits/f96d439308f880b46d6981c6bcbe89031d7376ec

So this was something you had changed locally and committed.

**Comment by abist** [comment]
Yeah, but I didn't touch any packages at all and assumed it's from main. Latest in main - https://github.com/microsoft/azuredatastudio/blob/main/extensions/notebook/yarn.lock#L1759

matches this.

**Comment by abist** [comment]
@Charles-Gagnon Actually, my bad, it was 1.5.0, not sure how it updated. I'll send a PR to remove that change.

**Comment by Charles-Gagnon** [comment]
Well, yeah it does because you just changed it 😛 

Before that it did not

https://github.com/microsoft/azuredatastudio/commit/71cbe3fbf039fd446e021e7c07edacd8b40639a3#diff-d35e050c89e7e09369150219f78ac0735d0a813600d723b4d493e19e376c5d1fR1759

It's probably a side effect of me bumping up the min version so isn't incorrect. I'm just saying that in the future please make sure not to include unrelated changes in PRs. Use separate PRs for stuff like this.

**Comment by Charles-Gagnon** [comment]
It's fine to leave it - I updated the min version to 1.51 anyways. I may have just missed updating the yarn.lock again after doing so.
